pkg/scheduler/uthelper/helper.go

```diff
@@ -168,9 +176,6 @@ func (test *TestCommonStruct) CheckAll(caseIndex int) (err error) {
 
 // CheckBind check expected bind result
 func (test *TestCommonStruct) CheckBind(caseIndex int) error {
-	if test.ExpectBindsNum != len(test.ExpectBindMap) {
```

Comment:
We'd better check it first to expose invalid setting , and if test.MinimalBindCheck is true, skip this check.

Reply:
When users want minimal check, they will not specify `ExpectBindMap`, so we can't check `ExpectBindMap` directly.

Reply:
modified.